data/meetings/d2-friends-of-bill-w.yaml

```diff
@@ -0,0 +1,14 @@
+area: 63
+district: 10
```

Comment:
Is it district 2 or 10?

Reply:
Now I know why you wanted me to put d2 in front of the other file. I will make changes, thanks.

Reply:
This is the only place any actual "logic" for districts exists - https://github.com/siouxfallsaa/siouxfallsaa.org/blob/master/Makefile#L21

Reply:
I suspect some are labeled District 5 which are not actually District 5 and this should import more districts, but I was unable to get any help sorting that stuff out.

Reply:
I can help go through and check districts as well. Which output should I put the parser on?

Reply:
I do not understand this question.

Reply:
The output can be selected to python, Json, or canonical? I selected python and it passed checks.